appcenter-crashes/scripts/postinstall.js

```diff
@@ -0,0 +1,50 @@
+const fs = require('fs');
+const path = require('path');
+
+const projectDirectory = path.resolve(`${__dirname}${path.sep}..${path.sep}..${path.sep}..`);
+const testDirectory = path.join(projectDirectory, 'test');
```

Comment:
This assume's the dependent's directory structure, and overall feel a little uneasy about "injecting" source files into a dependent.

What I do find interesting/appealing is the automatic updating of `packageJson.jest.setupFiles`. 

How about defining the mock file _in this project_ and link to it, rather than creating it in the user's directory?

Eg:
```js
projectJson.jest.setupFiles.push('node_modules/appcenter-crashes/jest/mocks.js')
```